mix.exs

```diff
@@ -1,13 +1,13 @@
 defmodule GenRMQ.Mixfile do
   use Mix.Project
 
-  @version "1.3.0"
+  @version "1.4.0"
```

Comment:
Not 100% sure but I think in terms of SemVer we have to bump the major version

Reply:
Then maybe we should consider bumping directly to 1.7 - what do you think? Is it too much to stop supporting two versions in one release :)?

Reply:
Let's go to 1.7 directly. Makes more sense to me.